woocommerce/woocommerce-android
# [JITM] Support flexible jitms so we can promote TTP better

Closes: #8696


### Description


Adds handling of the background and badge images provided from the backend.

Can be tested with this fluxc PR https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2698

### Testing instructions

* Either use proxy software to tweak JITMs response or modify the code with something like:

```
    val assets = mapOf(
         "background_image_url" to "https://w7.pngwing.com/pngs/308/74/png-transparent-computer-icons-setting-icon-cdr-svg-setting-icon.png",
         "badge_image_url" to "https://img.freepik.com/free-icon/internet_318-922355.jpg"
    )

     backgroundImage = assets.getBackgroundImage(), //line 91 JitmViewModel
     badgeIcon = assets.getBadgeIcon(),
```

Notice that provided images are displayed on the JITM.

* Run without response modification and notice that default (the same as now) images are used

### Images/gif


![image](https://user-images.githubusercontent.com/4923871/230571421-53f469c5-74c4-454f-b5a5-ff46b0da3df8.png)
